packages/account-v2/src/containers/PaymentMethods/PaymentMethodForm.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { TPaymentMethod, TPaymentMethodIdentifier, TPaymentMethodInfo } from 'src/types';
+import { PaymentMethodField } from './PaymentMethodField';
+import { PaymentMethodInstructions } from './PaymentMethodInstructions';
+
+type TPaymentMethodFormProps = {
+    paymentMethodDetail: TPaymentMethodInfo;
+};
+
+export const PaymentMethodForm = ({ paymentMethodDetail }: TPaymentMethodFormProps) => {
+    const paymentMethod = paymentMethodDetail.paymentMethod.toLowerCase() as TPaymentMethod;
+
+    return (
+        <div className='flex flex-col gap-24'>
+            <PaymentMethodInstructions paymentMethod={paymentMethod as TPaymentMethod} />
+            <fieldset className='flex flex-col gap-24'>
+                {paymentMethodDetail.items &&
+                    paymentMethodDetail.items.map(item => (
```

Comment:
Instead of 
```
 {paymentMethodDetail.items &&
                    paymentMethodDetail.items.map(item => (
```
cant we just` paymentMethodDetail?.items?.map(item => (..` directly